01-Login/.dockerignore

```diff
@@ -1,6 +1,2 @@
-package-lock.json
-node_modules
-.gitignore
-exec.sh
-exec.ps1
-README.md
+node_modules/
```

Comment:
I think `node_modules/` can be removed from the `.dockerignore` file, as Docker will also pay attention to whatever is in the `.gitignore` file. (I know the Vue sample might have this in, but that is to be changed)